ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.3.1f1
+m_EditorVersion: 2017.3.0f3
```

Comment:
This will happen everytime someone with a different version opens the project... it's to be expected .. as long as no one is using an ancient version it's fine.